image/templates/helm/stackrox-central/templates/02-config-controller-01-security.yaml

```diff
@@ -0,0 +1,41 @@
+{{- include "srox.init" . -}}
```

Comment:
I don't think the config-controller needs to run with the more privileged `nonroot-v2` SCC.